src/Config/RabbitMQOptions.cs

```diff
@@ -19,5 +16,9 @@ public class RabbitMQOptions
         public string ConnectionString { get; set; }
 
         public int Port { get; set; }
+
+        public PrefetchOptions PrefetchOptions { get; set; }

```

Comment:
nit: would be be great if you can fill in comments here too as these are public properties